Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -339,6 +354,7 @@ class AODProducerWorkflowDPL : public Task
   uint32_t mTrackCovOffDiag = 0xFFFF0000;      // 7 bits
   uint32_t mTrackSignal = 0xFFFFFF00;          // 15 bits
   uint32_t mTrackTime = 0xFFFFFFFF;            // use full float precision for time
+  uint32_t mTPCTime0 = 0xFFFFFC00;             // 13 bits
```

Comment:
with this you have in average `~0.3` TB (`~0.8` cm error), and in worst case 1 TB. Is not it too crude for anything useful?

Reply:
you are right, @miranov25 we talked about truncating this value a while ago, what precision do you need here? (I changed it now to 18bits mantissa -> 130um in z)

Reply:
Marian approved in private exchange.